src/OmniSharp.Abstractions/Models/v1/Completion/CompletionItem.cs

```diff
@@ -85,7 +85,7 @@ public class CompletionItem
         /// <summary>
         /// Index in the completions list that this completion occurred.
         /// </summary>
-        public int Data { get; set; }
+        public (int Index, bool IsExpanded) Data { get; set; }
```

Comment:
since we have a raw HTTP interface too, I think it would make more sense to capture it into a regular DTO

Reply:
Yeah I'm not sure how many users we have of the http interface at this point, but it is still supported at the moment.

Reply:
I take it ValueTuple does something weird with the HTTP interface?

Reply:
i don't think JSON.NET can serialize them https://github.com/JamesNK/Newtonsoft.Json/issues/1230

Reply:
I can actually just remove this addition and leave it as it was, will do that.